.github/workflows/ci.yml

```diff
@@ -46,6 +41,15 @@ jobs:
           - name: CPython 3.9
             tox: py39
             action: 3.9
+          - name: CPython 3.10
+            tox: py310
+            action: '3.10'
+        exclude:
+# Windows 10 not working yet, because there are no wheels yet for it.
+         - os:
+               name: Windows
+           python:
+               name: CPython 3.10
```

Comment:
Looks ok over at https://github.com/python-qt-tools/PyQt5-stubs/runs/4411387242?check_suite_focus=true from https://github.com/python-qt-tools/PyQt5-stubs/pull/180.  Note that the wheels are abi3 https://pypi.org/project/PyQt5/5.15.6/#files so they should be compatible with any new version of Python3.  Though, that was introduced in 5.15.5 which we only recently got updated past.

```suggestion
```

Reply:
Good that it is working. This was after my post to PyQt5 mailing list that Phil released new versions.